plasmapy/diagnostics/charged_particle_radiography/tests/test_detector_stacks.py

```diff
@@ -10,14 +10,20 @@
     Layer,
     Stack,
 )
-from plasmapy.utils.data.downloader import get_file
+from plasmapy.utils.data.downloader import Downloader
 
 
 @pytest.fixture()
-def hdv2_stack(tmp_path):
+def downloader(tmp_path):
+    # Turn off validation so API calls aren't used
+    return Downloader(directory=tmp_path, validate=False)
+
+
+@pytest.fixture()
```

Comment:
Oops, I thought this was what fixtures already did >< Yes that is good! 

I actually tried making this a `package` scope, so it can be used in other test folders, but that causes issues if you try to run those test files without this file. So, instead I've changed the function to use a session-scoped temporary directory, so tests that download files should leave them there, so that if another function in another test tries to get the same file, it doesn't have to re-download it.